lib/utils.nix

```
@@ -11,6 +11,12 @@ with lib;
     "__empty" = null;
   };
 
+  # Turn all the keys of an attrs to raw lua.
+  toRawKey = mapAttrs' (n: v: nameValuePair "__rawKey__${n}" v);
```

I think this should be a plural:

```suggestion
  toRawKeys = mapAttrs' (n: v: nameValuePair "__rawKey__${n}" v);
```

(also update usage in `mkRawKey`, of course)